lib/eventFactory.js

```diff
@@ -667,7 +768,12 @@ class EventFactory extends BaseFactory {
                     if (prRef) {
                         modelConfig.pr.ref = prRef;
 
-                        return p.syncPR(prNum);
+                        // return p.syncPR(prNum);
+                        if (subscribedEvent) {
+                            return p;
+                        }
+
+                        return p.syncPRs();
```

Comment:
wht did this change to `syncPRs` ?

Reply:
Yeah it was actually syncPRs. But if there is a subscribed event, there is no point in calling the syncPRs because the repository would not be a pipeline in screwdriver and hence should not affect the PR.